src/pymovements/dataset/_utils/_downloads.py

```diff
@@ -28,80 +28,10 @@
 from tqdm.auto import tqdm
 
 from pymovements._version import get_versions
-from pymovements.dataset._utils._archives import extract_archive
 
 USER_AGENT: str = f"pymovements/{get_versions()['version']}"
 
 
-def download_and_extract_archive(
```

Comment:
this function is not needed anymore. the functionality is implemented in `dataset_download.download_dataset()`.

this implementation is moved to the deprecated function `utils.download_and_extract_archive`